blog/_posts/Hola, Amigos.md

```diff
@@ -0,0 +1,97 @@
+---
+**title:** We solemnly swear we’re upto no good!
```

Comment:
Hi @avneet14027 ! You would need to use exact format as used in other files for declaring parameters. For example, no use of '**', permalink same as file name etc. Please check and test them locally.